Rakefile

```diff
@@ -49,8 +49,10 @@ CrossLibraries = [
 	['x64-mingw-ucrt', 'mingw64', 'x86_64-w64-mingw32'],
 	['x86-mingw32', 'mingw', 'i686-w64-mingw32'],
 	['x64-mingw32', 'mingw64', 'x86_64-w64-mingw32'],
-	['x86_64-linux', 'linux-x86_64', 'x86_64-linux-gnu'],
-	['aarch64-linux', 'linux-aarch64', 'aarch64-linux-gnu'],
+	['x86_64-linux-gnu', 'linux-x86_64', 'x86_64-linux-gnu'],
+	['x86_64-linux-musl', 'linux-x86_64', 'x86_64-unknown-linux-musl'],
+	['aarch64-linux-gnu', 'linux-aarch64', 'aarch64-linux-gnu'],
+	['aarch64-linux-musl', 'linux-aarch64', 'aarch64-linux-musl'],
```

Comment:
I wonder if we can omit `--with-toolchain`? I've never had to set this up as I thought `rake-compiler-dock` did this.

Reply:
I guess it's needed for OpenSSL. Without `--with-toolchain` I get:

```
install libcrypto.a -> /home/stanhu/github/ruby-pg/ports/x64-mingw32/lib64/libcrypto.a
/bin/sh: 5: -ranlib: not found
make: *** [Makefile:1987: install_dev] Error 127
```